modules/rosa-adding-tuning.adoc

```diff
@@ -11,8 +11,8 @@ You can add tunings for compute (also known as worker) nodes in a machine pool t
 .Prerequisites
 
 ifdef::openshift-rosa[]
-* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
-* You logged in to your Red Hat account by using the ROSA CLI.
+* You installed and configured the latest {product-title} (ROSA) CLI, `rosa`, on your workstation.
+* You logged in to your Red Hat account using the ROSA CLI (`rosa`).
 * You created a {product-title} (ROSA) cluster.
```

Comment:
On line 16, you don't need to use {product-title} (ROSA) Here because you've already expanded the product name in an earlier bullet. This can just be "You created a ROSA cluster."